src/views/index.html

```diff
@@ -124,6 +124,12 @@
   </div>
 </section>
 {# We could use a fractal to stylistically split this sections apart #}
+<section class="landing-quote-section" id="endorsement-quote">
+  <div class="content-wrapper-center">
+    <blockquote>Computer technology is a key driver underpinning our modern economy and society. Hack Cambridge provides an excellent platform for teams of student coders to compete and master their digital abilities right at the heart of the Cambridge technology cluster.</blockquote>
+    <cite><div>Professor Nigel K. H. Slater. Pro-Vice-Chancellor, Enterprise and Regional Affairs</div><img class="endorsement-logo" src="{{ asset('images/logos/university-of-cambridge-logo.svg') }}" alt="University of Cambridge logo"></cite>
```

Comment:
Pedant here, but can we make the alt text "University of Cambridge" and not add "logo"? Screen readers will then read out the citation + image just as we'd read it.